.github/workflows/release-cli.yml

```diff
@@ -112,6 +109,20 @@ jobs:
             build/constellation-${{ matrix.os }}-${{ matrix.arch }}.exe
             build/constellation-${{ matrix.os }}-${{ matrix.arch }}.exe.sig
 
+  upload-terraform-module:
```

Comment:
since you make it a separate job, the follow up jobs need to depend on it explicitly.

Reply:
The other jobs shouldn't depend on it, or do I miss something? My whole point was to make it run in parallel.

Reply:
`Download Terraform module` depends on the upload so, `provenance-subjects`,  `provenance-verify` and `release` need it, right?